lib/npg_pipeline/function/start_stop.pm

```diff
@@ -68,16 +67,12 @@ sub _token_job {
   $pipeline_name ||= q[];
   my $job_name = join q{_}, $subroutine_name, $self->label(), $pipeline_name;
 
-  my $d = npg_pipeline::function::definition->new(
-    created_by    => __PACKAGE__,
-    created_on    => $self->timestamp(),
+  my $d = $self->create_definition({
     identifier    => $self->label(),
     job_name      => $job_name,
     command       => '/bin/true',
     num_cpus      => [0],
-    queue         =>
-      $npg_pipeline::function::definition::SMALL_QUEUE,
-  );
+  });
```

Comment:
CPU num definition to be moved to the resource definition for the function, identifier is not needed.